### PR TITLE
Sadyk

### PR DESCRIPTION
**Pull request by Saadyk94**
API test case

### CODE REVIEW

File: sadyk/API-assignment/API test case [hunk 1]
```diff
@@ -0,0 +1,138 @@
+TEST CASE API
+URL: https://petstore.swagger.io/
+
+
+Test scenario ID - 1
```
Comment (Pokhilenko) [inline sadyk/API-assignment/API test case:5]: It's a good practice to provide short name to the test case/scenario

Comment (Pokhilenko) [reply sadyk/API-assignment/API test case:5]: something like "should successfully create new order"

File: sadyk/API-assignment/API test case [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+1. Copy URL and input in Postman: https://petstore.swagger.io/#/store/order
+2. Select "POST" hhtp methods from dropdown menu 
+3. Input in body:(
+    {
+  "id": 0,
+  "petId": 0,
+  "quantity": 0,
+  "shipDate": "2022-12-09T13:29:29.009Z",
+  "status": "placed",
+  "complete": true
```
Comment (Pokhilenko) [inline sadyk/API-assignment/API test case:45]: Would be super cool if we had some negative scenario for the request body as well, not only URL

File: sadyk/API-assignment/API test case [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+
+Test scenario ID - 3
+Test case description: Positive test case "GET"
+Pre-Condition: need Postman
+
+Test steps:
+1. Input URL in postman: https://petstore.swagger.io/v2/store/order/10
+2. Select "GET" hhtp methods from dropdown menu 
+3. Click "Send" button
```
Comment (Pokhilenko) [inline sadyk/API-assignment/API test case:64]: here, within this document we assume that user is familiar with postman, so we can remove extra steps like "Select RAW", "select GET", "click send" etc.

e.g. here we can make it using single step "Send GET request to https://petstore.swagger.io/v2/store/order/10 using Postman"

File: sadyk/API-assignment/API test case [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+Test steps:
+1. Input URL in postman: https://petstore.swagger.io/v2/store/order/1
+2. Select "GET" hhtp methods from dropdown menu 
+3. Click "Send" button
+
+Expected result: Status 404 not found
+Actual result: Status 404 not found
+
+
+
```
Comment (Pokhilenko) [inline sadyk/API-assignment/API test case:84]: too many blank lines, 2 would be enough

File: sadyk/API-assignment/API test case [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+  "shipDate": "2022-12-09T13:29:29.009Z",
+  "status": "placed",
+  "complete": true
+}
+)
+4. Select "Raw"
+5. Select "JSON" from formats dropdown menu
+6. Click "Send" button
+
+Expected result: Status 200 OK
```
Comment (Pokhilenko) [inline sadyk/API-assignment/API test case:26]: I believe there is also a response body expected, not only the status

**Review by Pokhilenko — CHANGES_REQUESTED** [review]

File: sadyk/API-assignment/API test case.txt [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+Test case description: Negative test case "GET"
+Pre-Condition: need Postman
+
+Test steps:
+1. Input incorrect URL in postman: https://petstore.swagger.io/v2/store/inventory1
+2. Select "GET" hhtp methods from dropdown menu 
+3. Click "Send" button
+
+Expected result:
+    "code": 404,
```
Comment (Pokhilenko) [inline sadyk/API-assignment/API test case.txt:217]: `{` is missing

**Review by Saadyk94 — COMMENTED** [review]
done

File: sadyk/API-assignment/API test case.txt (hunk 6 above)
Comment (Saadyk94) [reply sadyk/API-assignment/API test case.txt:218]: done

File: sadyk/API-assignment/API test case.txt [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+    "type": "unknown",
+    "message": "null for uri: http://petstore.swagger.io/v2/store/inventory1"
+}
+    Status 404 not found
+Actual result: 
+    "code": 404,
```
[the remark this answers is not in the document]
Comment (Saadyk94) [reply sadyk/API-assignment/API test case.txt]: done

**Pull request merged**